### DISCUSSION TITLE
user block

### DISCUSSION THREAD

**Discussion by mhaddaou** [vercel/community#4211] [Account Issue] [answered]
### Question

hello i hope you are fine i don't know why when i sing in with my github tell me user blocked

**Comment by hungchihsueh** [comment]
same, please help!

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

You can email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the support team. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by mhaddaou** [reply]
hi , thank you  about your response , i will contact them, thanks once again
